TerminalDocs/faq.yml

```diff
@@ -0,0 +1,94 @@
+  ms.date: 11/15/2021
+  ms.author: mattwoj
+  author: mattwojo
+  manager: jken
+  ms.reviewer: cinnamon
+    
+title: Windows Terminal FAQ
+summary: |
+  Find answers to some of the most frequently asked questions about Windows Terminal.
+
+sections:
+  - name: General
+    questions:
+      - question: How do I run a shell in Windows Terminal in administrator mode?
+        answer: |
+          To run Windows Terminal with elevated administrator permission (admin mode), right-click on the Windows Terminal icon, then again on the Windows Terminal title that displays, then select "Run as administrator".
+
+          ![Windows Terminal admin mode](./images/admin-mode.png)
+          
+          Alternatively, you can open the Windows Quick Access menu using the shortcut, ⊞ Windows key + X, and then selecting Windows Terminal (Admin).
+
+      - question: Is it possible to mix admin and non-admin tabs in a Windows Terminal window?
+        answer: |
+          No, mixing administrator-level permission tabbed shells with those that do not have elevated administrator permission is not supported due to security concerns.
+
+      - question: Can I use Windows Terminal as the integrated terminal in VSCode?
+        answer: |
+          No, Visual Studio Code is xtermjs and written in TypeScript while Windows Terminal is native code.
+
+      - question: What shells does Windows Terminal support?
+        answer: |
+          Windows Terminal will support any command line or shell that you have on your machine, including those that are included with Windows like PowerShell or Windows Command Prompt (cmd.exe), as well as any Linux distribution that can be installed with [WSL](/windows/wsl/), Azure Cloud Shell, Git Bash, etc. The terminal will automatically detect when you've installed a Linux distribution with WSL and create a profile for you. It can also save your Azure credentials, so you can quickly log in quickly to Azure Cloud Shell.
+
+      - question: How can I manually add a shell?
+        answer: |
+          In your [settings.json file](./install.md#settings-json-file), you can create or modify profiles that run any command-line executable. In the settings.json file, set "commandline" to whatever you want. For example, powershell --> "pwsh.exe".
+
+      - question: What is the difference between Windows Terminal and Windows Terminal (Preview)?
+        answer: |
+          Windows Terminal is [open-source](https://github.com/microsoft/terminal) and under active development. The recommended way to [install](./install.md) is via the Microsoft Store, which will provide automatic updates whenever they are released. [Windows Terminal](https://www.microsoft.com/p/windows-terminal/9n0dx20hk701?rtc=1&activetab=pivot:overviewtab) stable public release receives frequent updates, but the development team has also provided the [Windows Terminal Preview]() release for those interested in trying the latest features as they are developed.
```

Comment:
This reads initially as though the Preview build isn't open source? I'd probably reword it to just describe the difference between the two, rather than explaining what terminal is? Also, the link for Preview is missing. :)

Reply:
Fixed this answer -- reworded to more accurately reflect the difference and not include the open-source stuff.